### ISSUE TITLE
What to do about system

### ISSUE THREAD

**Issue by fdmalone** [JoonhoLee-Group/ipie#250]
This is just a tuple of the electron numbers so seems pretty redundant?

**Comment by linusjoonho** [comment]
We kept it this way because we thought system could include information about symmetry too. 
If we do k-points and space group symmetry, for instance, shouldn't that information be available through system?

**Comment by fdmalone** [comment]
What information are you thinking about? I would think the Hamiltonian would just have this information?

**Comment by linusjoonho** [comment]
Oh - is the list of k-points, for instance, a part of Hamiltonian?
Also, how about temperature?

**Comment by fdmalone** [comment]
I sort of feel like maybe these are just single parameters (temperature would be eaten by propagator for instance) and k-points which are a property of the Hamiltonian (they're in the definition of it, so it doesn't seem wrong right?) Anyway, not super important just popped into my head again.

**Comment by linusjoonho** [comment]
I like the idea of removing the class, but somebody must make the changes everywhere, right?

**Comment by fdmalone** [comment]
yeah, maybe next time I get motivated I will do it. It's not urgent and fairly easy to do. Just using github issues to think out loud lol.
